app/build.gradle

```diff
@@ -64,6 +64,7 @@ dependencies {
     implementation "androidx.navigation:navigation-ui-ktx:$rootProject.navigationVersion"
     implementation "androidx.recyclerview:recyclerview:$rootProject.recyclerViewVersion"
     implementation "androidx.room:room-runtime:$rootProject.roomVersion"
+    implementation "androidx.viewpager2:viewpager2:$rootProject.viewPagerVersion"
```

Comment:
OK to leave as is, but an explicit ViewPager2 dependency might help keep ViewPager2 pinned to the latest version (instead of what is bundled in MDC)